### ISSUE TITLE
https://github.com/grafeas/grafeas BUILD generation fails

### ISSUE THREAD

**Issue by c4milo** [bazelbuild/bazel-gazelle#918]
### What version of gazelle are you using?
0.22.1

### What version of rules_go are you using?
0.24.3

### What version of Bazel are you using?
3.5.0

### Does this issue reproduce with the latest releases of all the above?
yes

### What operating system and processor architecture are you using?
macOS/64

### What did you do?
I just tried to generate BUILD files for a project that depends on Grafeas. 

```
bazel run //:gazelle -- update-repos -from_file=go.mod -to_macro=deps.bzl%go_repositories -prune=true
```

go.mod
```
go 1.15

require (
	github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815
	github.com/grafeas/grafeas v0.1.6
	github.com/hashicorp/go-hclog v0.14.1
	github.com/rs/xid v1.2.1
	google.golang.org/grpc v1.32.0
)
```

### What did you expect to see?
No errors when running `bazel build //...`

### What did you see instead?

```
./tools/darwin/bazelisk build //...
ERROR: /private/var/tmp/_bazel_caguilar/d14951fe9a2b599877cab0d9de914f20/external/com_github_grafeas_grafeas/proto/v1beta1/grafeas_go_proto/BUILD.bazel:3:11: no such target '@com_github_grpc_ecosystem_grpc_gateway//runtime:runtime': target 'runtime' not declared in package 'runtime' defined by /private/var/tmp/_bazel_caguilar/d14951fe9a2b599877cab0d9de914f20/external/com_github_grpc_ecosystem_grpc_gateway/runtime/BUILD.bazel and referenced by '@com_github_grafeas_grafeas//proto/v1beta1/grafeas_go_proto:grafeas_go_proto'
ERROR: /private/var/tmp/_bazel_caguilar/d14951fe9a2b599877cab0d9de914f20/external/com_github_grafeas_grafeas/proto/v1beta1/grafeas_go_proto/BUILD.bazel:3:11: no such target '@com_github_grpc_ecosystem_grpc_gateway//utilities:utilities': target 'utilities' not declared in package 'utilities' defined by /private/var/tmp/_bazel_caguilar/d14951fe9a2b599877cab0d9de914f20/external/com_github_grpc_ecosystem_grpc_gateway/utilities/BUILD.bazel and referenced by '@com_github_grafeas_grafeas//proto/v1beta1/grafeas_go_proto:grafeas_go_proto'
INFO: Repository remote_java_tools_darwin instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_caguilar/d14951fe9a2b599877cab0d9de914f20/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository remotejdk11_macos instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_caguilar/d14951fe9a2b599877cab0d9de914f20/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
ERROR: Analysis of target '//grafeas:grafeas' failed; build aborted: Analysis failed
INFO: Elapsed time: 2.412s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (34 packages loaded, 216 targets configured)
    Fetching @local_config_xcode; fetching
    Fetching @org_golang_x_tools; fetching
make: *** [build] Error 1
```

**Comment by jayconrod** [comment]
Looks like a duplicate of #892. This would also be resolved by #890.

**Comment by c4milo** [comment]
Ah great! Thanks!
